query/language/match.feature

```diff
@@ -245,6 +245,29 @@ Feature: TypeQL Match Clause
       | label:person |
 
 
+  Scenario: 'owns' matches types that own the specified attribute type with their inheriting subtypes
```

Comment:
We didn't have such steps in `match`, that's why I encountered it in `define`. Three simple tests for three capabilities.